languages/LinkEdit.groovy

```diff
@@ -34,11 +34,30 @@ sortedList.each { buildFile ->
 
 	// create mvs commands
 	String member = CopyToPDS.createMemberName(buildFile)
-	
+	String overrideMember = member
+
+	/*
+	 * The below steps are to cater to the scenario when the LINK member name is different from
+	 * the load module name. 
+	*/ 
+    if (buildFile.toLowerCase().endsWith(".lnk")) {
```

Comment:
This should be independent of the file extension.
What about if we add a flag to "inspect the link member" for different outputMember name?